app/templates/index.html

```diff
@@ -47,13 +47,13 @@
             },
             {
               "link": "/g-cloud",
-              "title": "Find cloud technology and support",
-              "body": "eg web hosting or IT health checks",
+              "title": "Find cloud hosting, software and support" if show_new_g9_content else "Find cloud technology and support",
+              "body": "eg content delivery networks or accounting software" if show_new_g9_content else "eg web hosting or IT health checks",
             },
             {
               "link": "/crown-hosting",
-              "title": "Buy physical datacentre space",
-              "body": "eg for services that can’t be migrated to the cloud",
+              "title": "Buy physical datacentre space for legacy systems" if show_new_g9_content else "Buy physical datacentre space",
```

Comment:
We (well, @superroz) only just removed the "legacy systems" text, I think at the behest of CCS. Are we 100% sure we're putting it straight back in again?